### ISSUE TITLE
Series with a date from theTVDB and search issue in addic7ed

### ISSUE THREAD

**Issue by SkipCool33** [Diaoul/subliminal#628]
I've got an annoying bug with addic7ed
When a serie contains a date in its title in theTVDB, and not in addic7ed, then series are not found.

For example:
Castle (2009)
http://thetvdb.com/?tab=series&id=83462&lid=7
and
http://www.addic7ed.com/show/424

INFO:subliminal.refiners.tvdb:Searching series 'Castle'
DEBUG:subliminal.refiners.tvdb:Found 39 results
DEBUG:subliminal.refiners.tvdb:Found result with matching year
DEBUG:subliminal.refiners.tvdb:Found series {u'seriesId': u'75394', u'status': u'Continuing', u'added': u'2008-10-17 15:05:50', u'airsTime': u'10:00 PM', u'network': u'ABC (US)', u'rating': u'TV-PG', u'overview': u"Rick Castle is one of the world's most successful crime authors. But when his rock star lifestyle isn't enough, this bad boy goes looking for new trouble and finds it working with smart, beautiful Detective Kate Beckett. Inspired by her professional record and intrigued by her buttoned-up personality, Castle's found the model for his bold new character whether she likes it or not. Now with the mayor's permission, Castle is helping solve crime with his own twist.", u'lastUpdated': 1460123911, u'id': 83462, u'seriesName': u'Castle (2009)', u'firstAired': u'2009-03-09', u'addedBy': 3071, u'banner': u'graphical/83462-g20.jpg', u'networkId': u'', u'genre': [u'Comedy', u'Crime', u'Drama'], u'airsDayOfWeek': u'Monday', u'runtime': u'45', u'siteRating': 8.8, u'zap2itId': u'EP01085588', u'imdbId': u'tt1219024', u'aliases': [u'Castle 2009']}
[...]
WARNING:subliminal.providers.addic7ed:Series not found in show ids
ERROR:subliminal.providers.addic7ed:No show id found for u'Castle (2009)' ({'country': None, 'year': 2009})

Another example:
Once Upon a Time (2011)
http://thetvdb.com/?tab=series&id=248835&lid=7
http://www.addic7ed.com/show/1808

**Comment by Diaoul** [comment]
Please use latest develop. This shouldn't happen anymore.

**Comment by ratoaq2** [comment]
There are still some issues with the latest develop. Apart from some small logging issues and missing firstAired, tvdb refiner is not picking the correct series when the series name contains the year:

```
subliminal --debug --legendastv user pass download -f -l pt-BR -p legendastv -m 97 "Once.Upon.a.Time.S01E01.720p.HDTV.X264-DIMENSION.mkv
Collecting videos  [####################################]  100%  Once.Upon.a.Time.S01E01.720p.HDTV.X264-DIMENSION.mkvDEBUG:subliminal.cli:Collecting path Once.Upon.a.Time.S01E01.720p.HDTV.X264-DIMENSION.mkv
INFO:subliminal.core:Refining video with metadata
INFO:subliminal.core:Refining video with tvdb
INFO:subliminal.refiners.tvdb:Searching series 'Once Upon a Time'
DEBUG:subliminal.refiners.tvdb:Found 15 results
Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/__init__.py", line 859, in emit
    msg = self.format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 732, in format
    return fmt.format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 471, in format
    record.message = record.getMessage()
  File "/usr/lib/python2.7/logging/__init__.py", line 335, in getMessage
    msg = msg % self.args
TypeError: %d format: a number is required, not str
Logged from file tvdb.py, line 292
DEBUG:subliminal.refiners.tvdb:Found series {u'seriesId': u'', u'status': u'Ended', u'added': u'2008-11-26 17:53:49', u'airsTime': u'', u'network': u'PBS', u'rating': u'', u'overview': u"A children's show centering around a witch named Kerfumbly, who has kidnapped a librarian named Marion, and holds her prisoner in her dungeon. In each episode, Kerfumbly demands a different kind of story from Marion, who gathers materials from the dungeon, puts them into a machine, and out pops a book. Marion then reads the book to the audience while illustrations of the story are shown on the screen. This show was filmed in Omaha, Nebraska for Nebraska Public Television, and was shown widely in the late 1970's and early 1980's. ", u'lastUpdated': 1457589412, u'id': 83882, u'seriesName': u'Once Upon A Time', u'firstAired': u'', u'addedBy': 5697, u'banner': u'', u'networkId': u'', u'genre': [u'Children'], u'airsDayOfWeek': u'', u'runtime': u'15', u'siteRating': 7.2, u'zap2itId': u'', u'imdbId': u'', u'aliases': []}
ERROR:subliminal.core:Failed to refine video
Traceback (most recent call last):
  File "/home/user/workspace/git/subliminal/subliminal/core.py", line 542, in refine
    refiner_manager[refiner].plugin(video, **kwargs)
  File "/home/user/workspace/git/subliminal/subliminal/refiners/tvdb.py", line 314, in refine
    video.year = datetime.strptime(series['firstAired'], '%Y-%m-%d').year
  File "/usr/lib/python2.7/_strptime.py", line 325, in _strptime
    (data_string, format))
ValueError: time data '' does not match format '%Y-%m-%d'
```

It's picking `tvdb_id = 83882`
where the correct one is `tvdb_id=248835`

The correct one has `name = u'Once Upon a Time (2011)'` and `aliases = [u'Once Upon a Time 2011']`

**Comment by Diaoul** [comment]
Should be fixed with d5b4a4f226ef9fc28dcb14f8ac2b515e39d92414
